emily/handler/src/database/accessors.rs

```diff
@@ -434,7 +443,14 @@ pub async fn pull_and_update_withdrawal_with_retry(
             return Ok(entry);
         }
 
-        if !is_trusted_key && entry.status != Status::Pending {
+        // We don't want to add a new entry if the status is already accepted.
+        // Updates Accepted -> Accepted occurs usually due to RBF.
+        if update.event.status == StatusEntry::Accepted && entry.status == Status::Accepted {
+            return Ok(entry);
+        }
```

Comment:
What about withdrawals switching `Accepted` -> `Accepted`? Rbf can't be here.

Reply:
cc @djordon

Reply:
These updates are for transactions where sweep in deposits and/or sweep out withdrawals, so we can have the status of a withdrawal request going from `Accepted` to `Accepted` when the signers try to RBF an outstanding sweep transaction.

Reply:
Aaaand, we want to change an event in that case, yes?

Reply:
I think we want the same behavior that we have for deposits, where Emily ignores events when we're going from `Accepted` to `Accepted`.